ffi/wgpu.h

```diff
@@ -728,6 +734,34 @@ typedef struct {
   uint32_t array_layer_count;
 } WGPUTextureViewDescriptor;
 
+
```

Comment:
I wonder where these come from?

Reply:
I also thought it was strange that cbindgen would suddenly add those ...